src/app/components/shared/menu/button/button.component.scss

```diff
@@ -6,11 +6,11 @@ button {
   border: 0px;
 
   span {
-    color: $primary;
+    color: var(--theme-highlight);
   }
 
   &:hover span {
-    color: darken($primary, 15%);
+    filter: brightness(0.85);
```

Comment:
I would suggest not doing this. 
A filter is a specific thing, it affects the entire element, including text.

Why not simply darken the colour with the hsl function? https://stackoverflow.com/a/55330103/224512